docker_app/readme.md

```diff
@@ -0,0 +1,8 @@
+1. Custom image build:
+    ```bash
+    sudo docker build --build-arg site_dir=<path to Lektor dir> -f docker_app/Dockerfile .
```

Comment:
sudo should not be necessary, the configuration should allow the current user to use docker.

Reply:
This little-readme just `memo` for docker-interface developers.

Reply:
I got that; but we should not promote bad practices, so I would remove `sudo`.